core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -226,22 +226,17 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
     gcsDAO.listBillingAccounts(userInfo) map(RequestComplete(_))
 
   def getBillingProjectMembership(projectName: RawlsBillingProjectName): Future[PerRequestMessage] = {
-    val membershipsFuture = for {
-      resourceIdsWithPolicyNames <- samDAO.getPoliciesForType(SamResourceTypeNames.billingProject, userInfo)
-      projectDetailsByName <- dataSource.inTransaction { dataAccess => dataAccess.rawlsBillingProjectQuery.getBillingProjectDetails(Set(projectName)) }
+    val membershipFuture = for {
+      resourceIdsWithPolicyName <- samDAO.getResourcePolicies(SamResourceTypeNames.billingProject, projectName.value, userInfo)
```

Comment:
Yeah, I double checked this in the sam swagger page. This endpoint behaves more restrictively than the other endpoints and 403's for cases where the user has `User` level access to the billing project. We'll need to use the more generic endpoint to get what we need.